plugins/llvm/llvm_disasm.cpp

```diff
@@ -33,6 +63,40 @@ void initialize_llvm() {
 
 using pred_fun = std::function<bool(const llvm::MCInstrDesc&)>;
 
+#if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR <= 5
```

Comment:
use strict comparison that will hit only `3.4`. We do not support the `3.5` version (the ubuntu 3.5 is different from an upstream 3.5.). Basically all the ifdefs should only work for 2 versions and give `#error` for everything else.